site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -10,10 +10,10 @@
     <hr>
 
     <div class="dropdown-wrapper">
-        <div onclick='updateDropdownStates($(this).find(".dropdown-arrow").hasClass("down"), "today_polls_dropdown", "{{ base_url }}")' class="dropdown-bar">
+        <div  onclick="updateDropdownStates($(this).find('.dropdown-arrow').hasClass('down'), 'today_polls_dropdown', '{{ base_url }}')" class="dropdown-bar">
             <h2> Today's Polls </h2>
             <div class="dropdown-btn">
-                <i class="dropdown-arrow{{dropdown_states['today'] ? ' down' : ' right'}}"></i>
+                <i class="dropdown-arrow{{dropdown_states['today'] ? ' down' : ' right'}} fa fa-chevron-right"></i>
```

Comment:
With these changes, the `right` class no effect anymore, correct? Can we then simplify this to be `{{ dropdown_states['today'] ? 'down' }}` (and same as below for other categories).

Could then also remove the toggle on the `right` class from https://github.com/Submitty/Submitty/blob/FixChevronsDarkMode-Polls/site/public/js/polls-dropdown.js.

Reply:
Yep, that is correct.  Changes have been made.